config.json

```diff
@@ -1150,7 +1148,6 @@
         "algorithms",
         "arrays",
         "loops",
-        "mathematics",
```

Comment:
Might have kept "math" for the change exercise. It's really more searching than math, so it's OK.

Reply:
Yeah, I feel like the math part of change is a bit incidental to the exercise 👍